webui/react/src/components/kit/svgIcons/ColumnsIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import useUI from 'stores/contexts/UI';
+import { Status } from 'utils/colors';
+import { DarkLight, getCssVar } from 'utils/themes';
+
+const SVG: React.FC<{ color: string }> = ({ color }) => (
+  <svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      clipRule="evenodd"
+      d="M9 5H5V19H9V5ZM5 20H9H10H14H15H19C19.5523 20 20 19.5523 20 19V5C20 4.44772 19.5523 4 19 4H15H14H10H9H5C4.44772 4 4 4.44772 4 5V19C4 19.5523 4.44772 20 5 20ZM15 19H19V5H15V19ZM14 19V5H10V19H14Z"
+      fill={color}
```

Comment:
mmm -- consider using `currentColor` for these instead so they match the font color of the surrounding text. That also means that we don't have to keep track of the ui theme in the component.